snapcraft.yaml

```diff
@@ -819,6 +819,7 @@ parts:
       - --disable-vdi
       - --disable-vnc
       - --disable-xen
+      - --disable-xkbcommon
```

Comment:
Do we also need the key map files too?

Reply:
I don't know if the keymap files are needed/used by anything. Maybe @mihalicyn would know?